server/portal/apps/tickets/rtUtil.py

```diff
@@ -60,7 +60,7 @@ def replyToTicket(self, ticket_id, reply_text, files=[]):
     def hasAccess(self, ticket_id, user=None):
         if user and ticket_id:
             ticket = self.tracker.get_ticket(ticket_id)
-            if user in ticket.get('Requestors', '') or user in ticket.get('Cc', ''):
+            if user.lower() in map(str.lower, ticket.get('Requestors', '')) or user.lower() in map(str.lower, ticket.get('Cc', '')):
```

Comment:
Just checking, do we know that `Requestors` and `Cc` are always arrays coming from `rt`?

Reply:
In this api, `Requestors` and `Cc` are coming from ticket response via RT client. 
And [RT](https://github.com/bestpractical/rt/blob/c3b237c73b2d3d2c51e2d43535696692fe00fe15/lib/RT/Ticket.pm#L891) does store both those fields. 

On how RT originally got a user's email - from Andrew:
> RT tries to pull from TAS. But if the user made the account by just emailing the ticketing system itself, they make it. So in those cases, it would involve me manually changing the email in the rt system

Reply:
Right, but do we expect them to be a `list` always? e.g. I'm not sure the `map` logic would work if the value came back as a `string`

Reply:
good question, checking on that. Documentation is poor, reading through their code. Will adjust to string or list if I can't guarantee a list.

Reply:
Fixed the code, adjusted tests to work for string or list, retested. Also, added tests when the fields are not available.

RT [tests ](https://github.com/python-rt/python-rt/blob/master/tests/test_rest1.py#L229)in its code base assumes array. But, I could not found any other typed info on REST endpoint properties.